backend/coins/btc/handlers/handlers.go

```diff
@@ -383,13 +383,26 @@ func (handlers *Handlers) getUTXOs(*http.Request) (interface{}, error) {
 func (handlers *Handlers) getAccountBalance(*http.Request) (interface{}, error) {
 	balance, err := handlers.account.Balance()
 	if err != nil {
-		return nil, err
+		return struct {
```

Comment:
define one `var result struct {` and make instances below, so it's easier to see what the result can be in each case.

(At some point we should probably refactor this and make a resuable result struct)